components/header/index.css

```diff
@@ -8,9 +8,66 @@
   flex-basis: 60%;
   align-self: stretch;
   height: inherit;
+  width: 100%;
+  transition: transform 0.3s ease-in, border 0s ease-in 0.3s;
+  will-change: transform;
 }
 
 .header-item.header-area-brand {
   flex-grow: 0;
   flex-basis: 11.5rem;
 }
+
+.menu-toggle {
+  display: none;
+  height: 100%;
+  justify-content: center;
+  flex-direction: column;
```

Comment:
Why flex?

Reply:
Well, I don't think about it much. I just use the first thing that works for me. If you have better css styling please share code snippet. 🙂

Reply:
It's good enough for now